bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -222,7 +226,7 @@ spec:
                 resources:
                   limits:
                     cpu: 200m
-                    memory: 1024Mi
+                    memory: 1Gi
```

Comment:
**Note to reviewers:** Unit conversion was done automatically, it shouldn't cause a problem.